### ISSUE TITLE
Status Image

### ISSUE THREAD

**Issue by four2six** [Loderian/Activities#5]
I did add a status image, that's not clickable in the activities-shortcode.php, as i needed it. Probably not where it should be. Maybe an idea for you.

something like this....

```
            case 'status_image':
                $join_clickable =
                    '<img src="' . esc_attr( $text ) . '"     
                            alt="' . esc_attr( $alt_text ) . '"/>';
```

**Comment by Loderian** [comment]
So do you want there to be an image (if a user can join) or a text saying that the user cant join for some reason?

**Comment by four2six** [comment]
No, I needed it for an "overview" page where the user can see which activities he/she joined. I used the status_image on that. I didn't want them to join accidentally on that page, but having to open the 'original' page and to use the join_image to actually and intentionally join the activity.

**Comment by Loderian** [comment]
I see. I can add it to the shortcode, it should be another branch in the evalutaion of input then the join evalutaion. Do you want it to show if the activity has ended, is full, etc or just if the user is participating/not participating?

**Comment by four2six** [comment]
Yeah, i thought another branch would be better.
In my case just seeing the result as in the example was enough. obviously you could think of other circumstances where someone might need another of the options. While you're at it, adding something like status_text (for just showing a text as the current status) would probably be a good idea too. 
What i personally would be missing is function showing all current activities of a user or of a responsible, a list of all all events with the status, etc. But that would be an even further enhancement and a lot more work.

**Comment by Loderian** [comment]
I added this to the latest update too. Input is `joined` and `not_joined`.
As for showing all the activities for a user; I think its best that each activity can have their own images/display options, instead of supplying through shortcode. Makes it easier to see, manage and reuse the images/text for an activity. This would probably be its own edit page.
But this a lot more work as you said. And it quickly ends up in the backlog of large requested features.
